test/unit/user_test.rb

```diff
@@ -1,26 +1,23 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
-
   test '#github_url returns github url' do
-    assert User.new(github: 'jroes').github_url == 'https://github.com/jroes'
+    github_url = User.new(github: 'jroes').github_url
+    assert_equal 'https://github.com/jroes', github_url
   end
 
   test 'public scope should only return public users' do
     user           = users(:mockstar)
-    repo           = Repo.new
```

Comment:
Is there a good reason why this was saving a Repo when it was testing User.public_profile? Am I missing something?